lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -329,6 +437,7 @@ class TritonGPUAccelerateMatmulPass
     mlir::RewritePatternSet patterns(context);
 #ifdef USE_ROCM
     patterns.add<::BlockedToMFMA>(context);
+    patterns.add<::ChainDotAMD>(context);
```

Comment:
I think it's better to add something in `BlockedToMFMA` instead of creating a new pattern.